File: Ejercicio Pdf/Semana3/Practicamos/Python/Ejercicio_Practicamos_4.py
# Hallar el sueldo neto de un trabajador en base al sueldo base, sabiendo que se le descuenta por AFP 10% del sueldo neto y 5% de seguro de vida. Además, se le considera un bono extra de 85 soles por escolaridad.
sueldo_base = float(input("Ingrese el sueldo base: "))
bono_escolaridad = 85.0
descuento_afp = 0.10
descuento_seguro_vida = 0.05
sueldo_neto = sueldo_base + bono_escolaridad - (sueldo_base * descuento_afp) - (sueldo_base * descuento_seguro_vida)
print("El sueldo neto del trabajador es: S/.", sueldo_neto)

File: 02_EstructurasControl/Secuenciales/Semana3_Secuenciales/Practicamos/C#/Ejercicio_Practicamos_4/Ejercicio_Practicamos_4.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Ejercicio_Practicamos_4
{
    internal class Ejercicio_Practicamos_4
    {
        static void Main(string[] args)
        {
            //Hallar el sueldo neto de un trabajador en base al sueldo base, sabiendo que se le descuenta por AFP 10% del sueldo neto y 5% de seguro de vida. Además, se le considera un bono extra de 85 soles por escolaridad.
            Console.Write("Ingrese el sueldo base del trabajador: ");
            double sueldo = double.Parse(Console.ReadLine());
            double descuento_afp = sueldo * 0.10;
            double descuento_vida = sueldo * 0.05;
            double bono = 85;
            double sueldo_neto = sueldo - descuento_afp - descuento_vida + bono;
            Console.WriteLine("El sueldo neto del trabajador es: " + sueldo_neto);
            Console.ReadKey();
        }
    }
}
